feature-libs/asm/customer-360/components/asm-customer-promotion-listing/asm-customer-promotion-listing.component.html

```diff
@@ -3,6 +3,61 @@ <h4 class="cx-asm-customer-promotion-listing-heading-text">
     {{ headerText }}
   </h4>
 </div>
+<ng-container *ngIf="isCustomerCoupon">
```

Comment:
is more reasonable put these to asm-customer-customer-coupon component, seems asm-customer-promotion-listing component is a more common and base list component ?

Reply:
will explain you in slack

Reply:
consider to use `ng-content`

Reply:
`ng-content` works well. Thx a lot!